scripts/solenumgen/main.go

```diff
@@ -0,0 +1,180 @@
+// Command solenumgen generates golang enum bindings for solidity enums.
+// This is a workaround for abigen that doesn't generate enums.
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+	"golang.org/x/tools/imports"
+	"os"
+	"strings"
+	"text/template"
+)
+
+var (
+	flagInput   = flag.String("input", "", "input sol file path to generate the enums from")
+	flagPackage = flag.String("package", "", "package name to generate the enum in")
+	flagOutput  = flag.String("output", "", "output file path to write generated codes")
```

Comment:
Not sure if plural was intended
```suggestion
	flagOutput  = flag.String("output", "", "output file path to write generated code")
```